ui/admin-portal/src/app/components/util/candidate-search-card/candidate-search-card.component.html

```diff
@@ -57,6 +57,17 @@ <h5  *ngIf="isContextNoteDisplayed() && !isList">Search Specific</h5>
             </app-cv-preview>
           </ng-template>
         </ng-container>
+
+        <!-- PROGRESS -->
+        <ng-container *ngIf="isSubmissionList" ngbNavItem="progress">
```

Comment:
This should really be "isSubmissionList()" - see comment below on inappropriate use of "get"